browser/typescript/v2/react-app/src/ampli/index.ts

```diff
@@ -0,0 +1,839 @@
+    } else if (apiKey) {
+      this.amplitude = amplitude;
+      return this.amplitude?.init(apiKey, undefined, { ...DefaultOptions, ...options?.client?.options });
+    } else {
+      console.error("ERROR: ampli.load() requires 'environment', 'client.apiKey', or 'client.instance'");
+    }
+  }
+
+  /**
+   * Identify a user and set user properties.
+   *
+   * @param userId The user's id.
+   * @param properties The user properties.
+   * @param options Optional event options.
+   */
+  identify(
+    userId: string | undefined,
+    properties: IdentifyProperties,
+    options?: IdentifyOptions,
+  ) {
+    if (!this.isInitializedAndEnabled()) {
+      return;
+    }
+
+    const event: IdentifyEvent = {
+      event_type: SpecialEventType.IDENTIFY,
+      user_properties: properties,
+      user_id: userId || options?.user_id,
+      device_id: options?.device_id
+    };
+
+    if (event.user_id) {
+      this.amplitude?.setUserId(event.user_id);
+    }
+    if (event.device_id) {
+      this.amplitude?.setDeviceId(event.device_id);
+    }
+
+    const amplitudeIdentify = new amplitude.Identify();
+    if (properties != null) {
+      for (const [key, value] of Object.entries(properties)) {
+        amplitudeIdentify.set(key, value);
+      }
+    }
+    return this.amplitude?.identify(
+      amplitudeIdentify,
+      options,
+    );
+  }
+
+  setGroup(name: string, value: string | string[]) {
+    if (!this.isInitializedAndEnabled()) {
+      return;
+    }
+
+    this.amplitude?.setGroup(name, value);
+  }
+
+  /**
+   * Identify a group and set group properties.
+   *
+   * @param groupType The group type.
+   * @param groupName The group name.
+   * @param properties The group properties.
+   * @param options Optional event options.
+   */
+  groupIdentify(
+    groupType: string,
+    groupName: string | string[],
+    properties: GroupProperties,
+    options?: GroupOptions,
+  ) {
+    if (!this.isInitializedAndEnabled()) {
+      return;
+    }
+
+    const event: GroupEvent = {
+      event_type: SpecialEventType.GROUP_IDENTIFY,
+      group_properties: properties,
+      user_id: options?.user_id,
+      device_id: options?.device_id
+    };
+
+    if (event.user_id) {
+      this.amplitude?.setUserId(event.user_id);
+    }
+    if (event.device_id) {
+      this.amplitude?.setDeviceId(event.device_id);
+    }
+
+    const amplitudeIdentify = new amplitude.Identify();
+    if (properties != null) {
+      for (const [key, value] of Object.entries(properties)) {
+        amplitudeIdentify.set(key, value);
+      }
+    }
+    return this.amplitude?.groupIdentify(groupType, groupName, amplitudeIdentify, options);
+  }
+
+  /**
+   * Track event
+   *
+   * @param event The event to track.
+   * @param options Optional event options.
+   */
+  track(event: Event, options?: EventOptions) {
+    if (!this.isInitializedAndEnabled()) {
+      return;
+    }
+
+    return this.amplitude?.logEvent(
```

Comment:
We are deprecating `logEvent` in favor of `track` in next gen.
```suggestion
    return this.amplitude?.track(
```

Reply:
Also can we call `track(event, options)` directly instead of needing to reference type and properties?